specs/2024-10-30.yaml

```diff
@@ -2457,27 +2456,12 @@ components:
           description: |-
             The unique identifier for the payment that the transaction is associated with.
           example: "pay_1LaXpKGUcADgqoEMl0Cx0Ygg"
-        payment:
-          $ref: "#/components/schemas/Payment"
-        payment_details:
-          $ref: "#/components/schemas/PaymentDetails"
```

Comment:
removed payment details from the create transactions request. Transactions are many-to-one with payments, so it wouldnt make sense to specify payment details in the create transactions route